src/CircleEditor/CircleEditorDocument.ts

```diff
@@ -191,4 +195,333 @@ export class CircleEditorDocument extends Disposable implements vscode.CustomDoc
       }
     };
   }
+
+  /**
+   * Guess data's type (int or float)
+   */
+  private guessExactType(n: any) {
+    if (Number(n) % 1 === 0) {
+      return "int";
```

Comment:
Could you use single quote if there is no special reason? :)
(please fix all the occurrences in this file)

Reply:
I modified it!